frontend/appflowy_flutter/lib/mobile/presentation/mobile_home_page.dart

```diff
@@ -0,0 +1,46 @@
+import 'package:appflowy/startup/startup.dart';
+import 'package:appflowy/user/application/auth/auth_service.dart';
+import 'package:appflowy_backend/protobuf/flowy-folder2/workspace.pb.dart';
+import 'package:appflowy_backend/protobuf/flowy-user/protobuf.dart';
+import 'package:flutter/material.dart';
+
+class MobileHomeScreen extends StatelessWidget {
+  const MobileHomeScreen(
+    this.user,
+    this.workspaceSetting, {
+    super.key,
+  });
+  static const routeName = "/MobileHomeScreen";
```

Comment:
The reason why I define the route name within every screen is to build the connection between them.

For example, when navigating to `SignInScreen`, its route name will be `SignInScreen.routeName`. Thus, we can always know which page it leads to by clicking `SignInScreen` on `SignInScreen.routeName`. In any place we put the route name, we can immediately know which page it leads to.  

The approach you suggested has a downside; wrong/misspelling of the route name can make it difficult to locate the correct screen. For example, if we change the screen name to MobileHomeFirstScreen but forget to change the route name, it will cause unnecessary confusion in finding the target screen through its route name.

Reply:
It would be difficult to list all the supported router names if you define them separately.

Reply:
You can use 
```
@override
  Widget build(BuildContext context) {
  return MaterialApp(
    routes: {
      HomeScreen.routeName: (context) => HomeScreen(),
      DetailScreen.routeName: (context) => DetailScreen(),
    },
  );
}

```
Instead of
```
@override
  Widget build(BuildContext context) {
  return MaterialApp(
    routes: {
      '/': (context) => HomeScreen(),
      '/details': (context) => DetailScreen(),
    },
  );
}
```